src/components/Dropdown/dropdown.css

```diff
@@ -2,8 +2,55 @@
 @import "src/css/variables/fonts.css";
 @import "src/css/variables/colors.css";
 
-.y-dropdown .ms-Dropdown-label{
+.y-dropdown .ms-Dropdown-label {
   @extend text__mediumSub;
   font-weight: $fontWeight__bold;
   color: $textColor__secondary;
 }
+
+.y-dropdown .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown-caretDownWrapper {
+  height: 36px;
+  line-height: 3.3rem;
+}
+
+.y-dropdown .ms-Dropdown-title {
+  color: $textColor;
+  border-color: $borderColor;
+}
+
+.y-dropdown .y-dropdown__caretDown {
+  color: $color__river;
+}
+
+.y-dropdown .ms-Dropdown:hover .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown:active .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown:focus .ms-Dropdown-title {
+  color: $textColor;
+  border-color: $borderColor;
+}
+
+.y-dropdown .ms-Dropdown:active .ms-Dropdown-title {
+  border-color: $color__lake;
+}
+
+.y-dropdown .ms-Dropdown:focus .ms-Dropdown-title {
+  border-color: $color__lake;
+}
+
+/* NOTE: y-dropdown cannot be used here since the options are not a descendent
+ * of dropdown element
+ */
```

Comment:
Reaching down into the Fabric styles feels bad, but that's what we've been doing and I know you're just following the pattern.

Looks like we can pass a className prop to Dropdown if that feels better to you.

Reply:
the passed  in className unfortunately won't apply to the dropdown options since they are rendered in a separate layer. We could propose an additional className for options in Fabric?